src/test/java/org/broadinstitute/hellbender/utils/smithwaterman/SmithWatermanAlignerAbstractUnitTest.java

```diff
@@ -21,6 +21,8 @@
  * Implementations of a {@link SmithWatermanAligner} can be tested by subclassing this test and overriding
  * {@link #getAligner()} to return an instance of the class to be tested
  */
+
+@Test
 public abstract class SmithWatermanAlignerAbstractUnitTest extends BaseTest {
```

Comment:
After https://github.com/broadinstitute/gatk/pull/3475, this should extend `GATKBaseTest`